inc/cleantalk-settings.php

```diff
@@ -3268,6 +3270,14 @@ function apbct_settings__get__long_description()
                 '<a href="https://cleantalk.org/help/how-to-hide-website-field-in-wordpress-comments{utm_mark}" target="_blank">' . __('Learn more.', 'cleantalk-spam-protect') . '</a>'
             )
         ),
+        'comments__the_real_person' => array(
+            'title' => __('The Real Person Badge!', 'cleantalk-spam-protect'),
+            //HANDLE LINK
+            'desc'  => sprintf(
+                __('Plugin shows special benchmark for author of a comment or review, that the author passed all anti-spam filters and acts as a real person. It improves quality of users generated content on your website by proving that the content is not from spambots. %s', 'cleantalk-spam-protect'),
+                '<a href="https://cleantalk.org/help/the-real-person{utm_mark}" target="_blank">' . __('Learn more.', 'cleantalk-spam-protect') . '</a>'
```

Comment:
Please, use LinkConstructor class. And check what {utm_mark} means there.

Reply:
Done https://github.com/CleanTalk/wordpress-antispam/commit/9d74f99faaadb564eeccddacabf3511ae90c8fae